docs/flink-ddl.md

```diff
@@ -144,44 +144,61 @@ CREATE DATABASE iceberg_db;
 USE iceberg_db;
 ```
 
-### `CREATE TABLE`
+## `CREATE TABLE`
 
 ```sql
 CREATE TABLE `hive_catalog`.`default`.`sample` (
     id BIGINT COMMENT 'unique id',
-    data STRING
-);
+    `data` STRING NOT NULL
+) 
+WITH ('format-version'='2');
 ```
 
 Table create commands support the commonly used [Flink create clauses](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/create/) including:
 
-* `PARTITION BY (column1, column2, ...)` to configure partitioning, Flink does not yet support hidden partitioning.
```

Comment:
why delete this?

Reply:
there is no partition by in the above example